oic.r.acl2.raml

```diff
@@ -52,6 +68,7 @@ traits:
                     "permission": 24
                   },
                   {
+                    "aceid": 2,
                     "subject": {
                       "uuid": "e61c3e6b-9c54-4b81-8ce5-f9039c1d04d9"
```

Comment:
There's a long-standing issue with the redundancy of "subjectuuid" and the use of a "uuid" element under the "subject" field. It hasn't happened everywhere. But in the discussion for CR-48 it was decided that there was no need to retain this redundancy (for backwards compatibility) since this is an entirely new resource.